src/map/lua/lua_baseentity.cpp

```diff
@@ -10361,7 +10361,8 @@ inline int32 CLuaBaseEntity::addStatusEffectEx(lua_State *L)
         (uint16)lua_tointeger(L, 5),
         (n >= 6 ? (uint16)lua_tointeger(L, 6) : 0),
         (n >= 7 ? (uint16)lua_tointeger(L, 7) : 0),
-        (n >= 8 ? (uint16)lua_tointeger(L, 8) : 0));
+        (n >= 8 ? (uint16)lua_tointeger(L, 8) : 0),
+        (n >= 9 ? (uint64)lua_tointeger(L, 9) : 0));
```

Comment:
wheel we're here lets take the opportunity to comment the arguments so we know both what the new one is and what the existing ones are.

@ibm2431 It may be worth doing what was done to `addItem()` and overloading a version that accepts a table. Unsure if that causes new problems on the arm64 build though (we should have somebody try the table version of `addItem()`.

Reply:
I will start commenting these.

Reply:
Ok this is done ✔